CIP-????/README.md

```diff
@@ -0,0 +1,132 @@
+---
+CIP: ?
+Title: Beacon Tokens & Distributed Dapps
+Category: Tools
+Status: Informational
+Authors:
+    - fallen-icarus <modern.daidalos@gmail.com>
+    - zhekson1 <zhekson@nomadpool.io>
+Implementors: []
+Discussions:
+Created: 2023-02-21
+License: CC-BY-4.0
+---
+
+## Abstract
+In the absence of atomic delegation, in order for layer 1 (L1) Dapp users to maintain full delegation control of their assets, each user must have his/her own address while using the Dapp. This poses a challenge: if all users have their own addresses while using the Dapp, how can users find and interact with each other, without relying on a central order batcher? This informational CIP proposes an NFT standard, called ***Beacon Tokens***, to solve this broadcasting issue. Using beacon tokens, it is possible to create distributed L1 Dapps (eg, DEXs, p2p lending, etc) where users not only maintain full delegation control of their assets, but can also maintain full custody of their assets at all times. Beacon tokens can be generalized to other use cases, such as creating an on-chain personal address book tied to - and fully recoverable by - a user's payment pubkey, or trustlessly sharing reference scripts with other blockchain users.
+
+## Motivation: why is this CIP necessary?
+To date, there are very few L1 Dapps in which users maintain full delegation control of their assets. Those that do, tend to be closed source and, therefore, do not share how they have overcome the broadcasting issue associated with giving each user his/her own Dapp address. The rest of the L1 Dapps usually end up pooling user assets together and therefore are forced to delegate the assets all together. In Proof-of-Stake (PoS) blockchains, this is a significant security concern. The more popular a L1 Dapp becomes, the more centralized the underlying stake becomes. Some Dapps try to address this concern by:
+
+1. Fractionalizing the asset pools - instead of all assets being at one address and delegated to only one stake pool, the assets are split among several addresses and delegated to several separate stake pools.
+2. Issuing governance tokens - each user gets a vote on where the pooled assets will be delegated.
+
+This general pattern is utilized by all kinds of L1 Dapps: DAOs, DEXs, p2p lending, etc. Unfortunately, the above mitigations do not fully solve the issue; the blockchain is still significantly more centralized than if full delegation control was used.
+
```

Comment:
I believe it's relevant that the huge community support for at least the non-custodial DEX applications that would be facilitated by Beacon Tokens should be accumulated in:

- _definitely_ the PR discussion
- _possibly also_ as references here in the Motivation section... once you can decide which make the best statements about how / how much the community might need this.

@AntMeyer1 at the CF also offered to help with this, but mainly it's your responsibility as authors to include these references as advocacy for your proposal.  So far I've bookmarked (cc @KtorZ):

- https://twitter.com/zhekree/status/1632042865013432323
- https://github.com/cardano-foundation/CIPs/pull/418#pullrequestreview-1325236056
- https://matrix.to/#/!QvtUklhUfmEuCBIexz:matrix.org/$lU0OwYOAQXpNYVFJl1I33YPhhQ_wH14Jzu5etQP9Drk?via=matrix.adamantium.online&via=matrix.org&via=forum.balanceanalytics.io

Reply:
Continuing this request in https://github.com/cardano-foundation/CIPs/pull/418#pullrequestreview-1398256071 because it appears to relate to all use cases that might also avail of Beacon Tokens.  TL;DR these issues seems so scattered across the DeFi field that someone might have to form a Working Group to clear out a path forward.